.gitlab/functional_test/regression_detector.yml

```diff
@@ -103,7 +117,8 @@ single-machine-performance-regression_detector:
     - "# beneath the error message in the GitLab CI logs                               #"
     - "#################################################################################"
     - echo ""
-    - dd-package --bucket binaries.ddbuild.io --package devtools/pr-commenter
+    # Pinned version of pr-commenter taken from https://github.com/DataDog/dogweb/blob/b2a891c2a2186b8fecd27ca9b13ebabe5f289a4a/tasks/gitlab/k8s-diff-helper.sh#L22
+    - dd-package --bucket binaries.ddbuild.io --package devtools/pr-commenter --distribution "20.04" --version "14692290-a6440fd1"
```

Comment:
Pinning the `pr-commenter` version seems to work for the time being, but it's likely the `pr-commenter` version will need to be updated because the CI Interfaces team has been adding telemetry to that service. [Recent versions of `pr-commenter`](https://github.com/DataDog/devtools/tree/fbb16979b1c97ed1a55e60b8a9e3acecc2728ddf/deb/pr-commenter) require a GitHub App key. For provenance purposes, it could make more sense for Agent to own that key.

Reply:
re: this specific line, without this change, the regression detector is definitely broken due to upstream changes in `pr-commenter`, so I'd be inclined to defer more `pr-commenter` changes to a later PR.